### PR TITLE
[Term Entry] Go Strings `Compare()`

### PR DESCRIPTION
**Pull request by manviii27**
### Description

 Added term entry for Go String function `Compare()` . File Location: `docs/content/go/concepts/string-functions/terms/compare/compare.md`

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [✓ ] All writings are my own.
- [✓ ] My entry follows the Codecademy Docs style guide.
- [ ✓] My changes generate no new warnings.
- [ ✓] I have performed a self-review of my own writing and code.
- [ ✓] I have checked my entry and corrected any misspellings.
- [ ✓] I have made corresponding changes to the documentation if needed.
- [ ✓] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ ✓] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ✓] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/go/concepts/strings/terms/compare/compare.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  - 'Strings'
+  - 'Functions'
+  - 'Characters'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+The **`strings.Compare()`** function is a built-in function in Golang that compares two strings lexicographically. It returns an integer value that indicates the relationship between the two strings.
+- If the two strings are equal, the function returns `0`.
```
Comment (SSwiniarski) [inline content/go/concepts/strings/terms/compare/compare.md:16]: Like to move bulleted list to end of syntax section.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@manviii27 There was an error in the path originally listed in the issue, so I fixed it to the correct path for you. Other than that, I just had one suggestion on first review.

File: content/go/concepts/strings/terms/compare/compare.md (hunk 1 above)
Comment (manviii27) [reply content/go/concepts/strings/terms/compare/compare.md]: @SSwiniarski I've made the changes, please check. Thank you.

**Review by SSwiniarski — APPROVED** [review]
@manviii27 Looks good for a second review.

File: content/go/concepts/strings/terms/compare/compare.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+The **`strings.Compare()`** function is a built-in function in Golang that compares two strings lexicographically. It returns an integer value that indicates the relationship between the two strings.
+
+## Syntax
+
+```pseudo
+strings.Compare(str1, str2)
+```
+
+Where `str1` and `str2` are the two strings that we want to compare.
```
Comment (yangc95) [inline content/go/concepts/strings/terms/compare/compare.md:23]:
```suggestion
Where `str1` and `str2` are the two strings to be compared.

```

File: content/go/concepts/strings/terms/compare/compare.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+  result1 := strings.Compare(firstString, secondString)
+  result2 := strings.Compare(firstString, thirdString)
+  result3 := strings.Compare(secondString, thirdString)
+
+  fmt.Println("The result of comparing 'Hello' and 'World' is:", result1)
+  fmt.Println("The result of comparing 'Hello' and 'Hello World' is:", result2)
+  fmt.Println("The result of comparing 'World' and 'Hello World' is:", result3)
+}
+
```
Comment (yangc95) [inline content/go/concepts/strings/terms/compare/compare.md:84]:
```suggestion
```

File: content/go/concepts/strings/terms/compare/compare.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+```pseudo
+strings.Compare(str1, str2)
+```
+
+Where `str1` and `str2` are the two strings that we want to compare.
+- If the two strings are equal, the function returns `0`.
+- If the first string is lexicographically less than the second string, the function returns a `negative` value.
+- If the first string is lexicographically greater than the second string, the function returns a `positive` value.
+
```
Comment (yangc95) [inline content/go/concepts/strings/terms/compare/compare.md:27]: preface this list

**Review by yangc95 — COMMENTED** [review]
@manviii27 looks great with a few style and structure corrections

File: content/go/concepts/strings/terms/compare/compare.md (hunk 4 above)
Comment (manviii27) [reply content/go/concepts/strings/terms/compare/compare.md:28]: @yangc95 I've made the changes, please review. Thank you

File: content/go/concepts/strings/terms/compare/compare.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+The **`strings.Compare()`** function is a built-in function in Golang that compares two strings lexicographically. It returns an integer value that indicates the relationship between the two strings.
+
+## Syntax
+
+```pseudo
+strings.Compare(str1, str2)
+```
+
+Where `str1` and `str2` are the two strings to be compared.
```
Comment (yangc95) [inline content/go/concepts/strings/terms/compare/compare.md:23]:
```suggestion
Where `str1` and `str2` are the two strings to be compared:
```

**Review by yangc95 — APPROVED** [review]
Great! LGTM - merging this in a bit

**Pull request merged**